lib/infrastructure/player_repository.dart

```diff
@@ -18,8 +18,8 @@ class _PlayerRepository extends PlayerController {
   }
 
   @override
-  void setSilentState(bool value) => _isSilent = value;
+  void setIsSound(bool value) => _isSilent = value;
```

Comment:
We can switch _isSilent to isSound here as well

Reply:
Sure.

Reply:
Updated.